### ISSUE TITLE
 Unhandled Exception: type '_InternalLinkedHashMap<String, dynamic>' is not a subtype of type 'String'

### ISSUE THREAD

**Issue by wrteam02** [Oni22/fluttermollie#2]
just run the demo code and got error like this..
![mollie](https://user-images.githubusercontent.com/79300515/110284989-97cfbe00-8008-11eb-915d-6ee6affb6c65.png)

**Comment by Bharavi26** [comment]
facing same issue..

please update that so i can use library without error....


![mollie](https://user-images.githubusercontent.com/36761436/110595318-e4033580-81a3-11eb-930d-971677336c11.png)

**Comment by Bharavi26** [comment]
hello, any update or anyone facing same issue ?

**Comment by markfili** [comment]
yes, the Mollie API has changed since last this project was upgraded. you can try this pull request https://github.com/Oni22/fluttermollie/pull/6, and confirm if the problem is fixed.
